src/pzt/admission-controller copy.yaml

```diff
@@ -0,0 +1,7 @@
+slug: kubernetes-admission-controller
+name: Kubernetes Admission Controller
+template: ingress
```

Comment:
if this is going to share a template with kubernetes ingress, let's rename the template to k8s, or something?